README.md

```diff
@@ -229,11 +229,14 @@ The following properties are expected to be present on the deployment `context`
 
 ## Configuring Amazon S3
 
-### Minimum S3 Permissions
+### Deployment user and S3 permissions
 
-Ensure you have the minimum required permissions configured for the user (accessKeyId). A bare minimum policy should have the following permissions:
+The environment in which the `ember deploy` command is run (outside of development) needs to have an AWS account with a policy that allows writing to the S3 bucket.
```

Comment:
the only note I'd make here is to remove `(outside of development)`, I believe that especially in early stages a lot of people will deploy from their own laptop so I think this could be confusing